lib/tesla/middleware/meta_logger.ex

```diff
@@ -71,6 +74,7 @@ if Code.ensure_loaded?(Tesla) do
       |> maybe_put_default_value(:log_level, :info)
       |> maybe_put_default_value(:log_tag, __MODULE__)
       |> maybe_put_default_value(:max_entry_length, :infinity)
+      |> maybe_put_default_value(:slicer, Slicer.DefaultImpl)
```

Comment:
🧵  Here we add a default Slicer to maintain backwards compatability, while allowing overriding the default implementaton.